pipeline.yaml

```diff
@@ -0,0 +1,24 @@
+description: |
+  A minimal pipeline that runs shear-specific tasks, separate from the
+  standard DRP processing.
+
+tasks:
+  makeWarp: descwl_coadd_task.MakeShearWarpTask
+  assembleCoadd:
+    # The configuration is same as the default values defined in the
+    # PipelineTasks. The purpose of overrides here is to demonstrate the
+    # configurable capacity of the pipeline and the tasks.
+    class: descwl_coadd_task.AssembleShearCoaddSlowTask
+    config:
+      connections.multiple_cell_coadd: shearCoadd
+      psf_dimensions: 21
```

Comment:
```suggestion
      psf_dimensions: 53
```

Reply:
Let's make the example override bigger since this bit might get copied places by accident.

Reply:
The point here was to show how to set configs in the pipeline. I'd rather remove this line entirely if you think this will get misused.

Reply:
sure let's remove it then